pkgs/applications/audio/clementine/default.nix

```diff
@@ -72,7 +72,8 @@ let
     name = "clementine-free-${version}";
     inherit src patches nativeBuildInputs postPatch;
 
-    buildInputs = buildInputs ++ [ makeWrapper ];
+    # gst_plugins needed for setup-hooks
+    buildInputs = buildInputs ++ [ makeWrapper gst_plugins ];
```

Comment:
shouldn't this be in the `buildInputs` array defined in the `let` block? Now `gst_plugins` is only used in the free derivation, but not in the unfree one

Reply:
No, the unfree derivation only builds the spotify blob, it doesn't rebuild the entire app.  See [line 109](https://github.com/NixOS/nixpkgs/pull/42314/files/6fcee4e1dde50336a7f1874dc63ec8e52b6254d9#diff-89f3ebf76e2a92d9e25c73c53cc06b55R109).

I've confirmed that this fixes the issue in `clementineUnfree` as well.

Reply:
I mean ... it wouldn't *hurt* to put it up there, I guess.  Maybe that's simpler?  But I don't think it's necessary for correctness.

Reply:
Any good reason why this PR has stalled?

Reply:
@Ma27 if you want me to amend my approach as you suggest, just LMK; easy to do.